### ISSUE TITLE
Unloadable save

### ISSUE THREAD

**Issue by 1xev3** [studio-minus/ppg-bug-reports#102]
**Describe the bug**
Inability to spawn a saved building

**Clearly describe how to reproduce the bug**
I loaded my old building, and changed the activating wires to destructible analogs, saved and after that my building stopped loading. I click on it and try to spawn, but when I spawn, another building appears, which I clicked on before.

**Mod list**
zooi-Signals and Logic Gates-2504621801 v1.2 for game v1.18.1
Currently ACTIVE

zooi-Per Pixel Destruction-2314372940 valpha 7 for game v1.13.1
Currently ACTIVE

Pump3d-Quick Right Click Options-2516131949 v1.0 for game v1.15
Currently ACTIVE

## System info
 - **OS**: Windows 10  (10.0.19042) 64bit
 - **CPU**: AMD Ryzen 5 2600 Six-Core Processor 
 - **GPU**: NVIDIA GeForce GTX 1650 SUPER
 - **Memory**: 16 GB
 
## LOGS
 ```
617,8102s [LOG] Action drag is in universe All
617,8102s [LOG] Action delete is in universe All
617,8102s [LOG] Action activateDirect is in universe Game
617,8102s [LOG] Action slowmo is in universe Game
617,8102s [LOG] Action toggleLimbStatus is in universe Game
617,8102s [LOG] Action time is in universe Game
617,8102s [LOG] Action tool1 is in universe Game
617,8102s [LOG] Action tool2 is in universe Game
617,8102s [LOG] Action tool3 is in universe Game
617,8102s [LOG] Action tool4 is in universe Game
617,8102s [LOG] Action tool5 is in universe Game
617,8102s [LOG] Action tool6 is in universe Game
617,8102s [LOG] Action tool7 is in universe Game
617,8102s [LOG] Action tool8 is in universe Game
617,8102s [LOG] Action tool9 is in universe Game
617,8102s [LOG] Action spawnLeft is in universe Game
617,8102s [LOG] Action spawnRight is in universe Game
617,8102s [LOG] Action left is in universe All
617,8102s [LOG] Action right is in universe All
617,8102s [LOG] Action fast is in universe All
617,8102s [LOG] Action undo is in universe All
617,8102s [LOG] Action snap is in universe All
617,8102s [LOG] Action snapToCenter is in universe Game
617,8102s [LOG] Action copy is in universe All
617,8102s [LOG] Action paste is in universe All
617,8102s [LOG] Action context is in universe All
617,8102s [LOG] Action pause is in universe All
617,8102s [LOG] Action toybox is in universe All
617,8102s [LOG] Action toolPowerToggle is in universe Game
617,8102s [LOG] Action pan is in universe All
617,8102s [LOG] Action panRight is in universe All
617,8102s [LOG] Action panLeft is in universe All
617,8102s [LOG] Action panUp is in universe All
617,8102s [LOG] Action panDown is in universe All
617,8102s [LOG] Action zoomIn is in universe All
617,8102s [LOG] Action zoomOut is in universe All
617,8102s [LOG] {
  "DropOnDeath": true,
  "LimbCrushing": true,
  "LimbOverflowExplosion": false,
  "TracerBullets": true,
  "FancyEffects": false,
  "ExpandedGore": false,
  "Decals": true,
  "ClampVolume": true,
  "ShowOutlines": true,
  "LogDebugMessages": false,
  "ShowFramerate": true,
  "Lighting": true,
  "VSync": true,
  "SMAA": true,
  "ZoomSensitivity": 1.0,
  "MasterVolume": 0.6013389,
  "SfxVolume": 0.5003042,
  "UserInterfaceVolume": 0.502189755,
  "AmbienceVolume": 0.09013651,
  "ShakeIntensity": 1.0,
  "SlowMotionSpeed": 10.1368208,
  "CrushForceMultiplier": 1.2,
  "DeleteWireByContextMenu": false,
  "SimplifiedBackground": false,
  "AmbientTemperatureTransfer": true,
  "TemperatureUnit": 0,
  "WindowMode": 1,
  "CollisionQuality": 1,
  "TonemappingMode": 0,
  "BloomMode": 2,
  "Brightness": 100.0,
  "PhysicsIterations": 16,
  "RenderScale": 1.0,
  "FramerateLimit": 144,
  "Resolution": null,
  "ShowTutorial": false,
  "ShowModLoadingFreeze": false,
  "RejectShadyCode": true
}
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \u18AE was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \u796D was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \u18AE was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \u2BAB was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uCAAE was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \u296D was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
617,8102s [WARNING] The character with Unicode value \uFFFD was not found in the [arial SDF] font asset or any potential fallbacks. It was replaced by Unicode character \u25A1 in text object [desc].
```

**Comment by mestiez** [comment]
I have the exact same mods you have and did exactly as you instructed, but I can load the save just fine. Could you send me the contraption files?

**Comment by mestiez** [comment]
I am pretty sure that the bug has been fixed thanks to #104
